composer.json

```diff
@@ -10,17 +10,18 @@
     "license": "MIT",
     "require": {
         "php": "^8.1",
-        "spiral/core": "^3.0",
+        "nyholm/psr7": "^1.5.0",
```

Comment:
Are you sure that requirement should not be dev?

Reply:
Moved to the require-dev